control-plane/connect-inject/webhook/mesh_webhook_ent_test.go

```diff
@@ -40,8 +41,7 @@ func TestHandler_MutateWithNamespaces(t *testing.T) {
 	}
 	s := runtime.NewScheme()
 	s.AddKnownTypes(schema.GroupVersion{Group: "", Version: "v1"}, &corev1.Pod{})
-	decoder, err := admission.NewDecoder(s)
-	require.NoError(t, err)
+	decoder := admission.NewDecoder(s)
```

Comment:
This was a breaking change in controller-runtime that you will see throughout this PR